springboot/template.native.yaml

```diff
@@ -8,7 +8,7 @@ Globals:
   Function:
     Tracing: Active
     CodeUri: target/function-native-zip.zip
-    Handler: org.springframework.cloud.function.adapter.aws.SpringBootStreamHandler
+    Handler: org.springframework.cloud.function.adapter.aws.FunctionInvoker::handleRequest
```

Comment:
`FunctionInvoker` is the new recommended way of invoking Spring Cloud Functions in AWS Adapter. `SpringBootStreamHandler` [should be deprecated](https://github.com/spring-cloud/spring-cloud-function/pull/898) and is already removed in Spring Cloud Function 4.0 SNAPSHOT